version/version.go

```diff
@@ -0,0 +1,13 @@
+package version
+
+import "strconv"
+
+type Version struct {
+	Major uint64
+	Minor uint64
+	Patch uint64
```

Comment:
```suggestion
	Major uint
	Minor uint
	Patch uint
```

Use simple `uint` types.